### PR TITLE
add: reporter-version

### PR DESCRIPTION
**Pull request by iurev**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -74,7 +78,13 @@ branding:
   icon: 'percent'
 runs:
   using: 'composite'
+
   steps:
+    - id: reporter-version
```
Comment (iurev) [inline action.yml:83]: Hi @nebolsin! I'm working on https://linear.app/coveralls/issue/SR-69/ability-to-select-version-of-coverage-reporter-public-issue-170. What do you think about using other actions in our workflows?

**Comment by mrexox** [comment]
Please add a note that it won't work for macOS runners 🙏

**Comment by iurev** [comment]
> Please add a note that it won't work for macOS runners 🙏

@mrexox, thank you! What do you think about using release numbers for brew: https://stackoverflow.com/questions/3987683/homebrew-install-specific-version-of-formula
`brew install postgresql@8.4.4`

**Comment by mrexox** [comment]
> > Please add a note that it won't work for macOS runners 🙏
> 
> @mrexox, thank you! What do you think about using release numbers for brew: [stackoverflow.com/questions/3987683/homebrew-install-specific-version-of-formula](https://stackoverflow.com/questions/3987683/homebrew-install-specific-version-of-formula) `brew install postgresql@8.4.4`

As long as I know they won't work for coveralls [tap](https://github.com/coverallsapp/homebrew-coveralls). This can probably be fixed with adding formulas for all versions but I have big concerns.

**Pull request closed without merging**